### DISCUSSION TITLE
package tidak terbaca

### DISCUSSION THREAD

**Discussion by beladinaaja** [teamalgoritma/community#707] [Q&A] [answered]
<img width="958" alt="Screenshot 2023-06-28 221742" src="https://github.com/teamalgoritma/community/assets/137804030/527ff175-a7b7-4aa2-9b1c-b1ccaf6f0e48">
Ini kenapa gak bisa ya kak

**Comment by mchen899** [comment]
ini apakah library nya tidak terbaca?
apakah seluruh package yang ada pada library tersebut sudah diinstall?

**Comment by kevbow** [comment, marked as answer]
Dear @beladinaaja,

Pada tampilan di screenshot ini tidak terdapat error, hanya warning saja. Hal ini tidak ada masalah ya Bu kecuali memang terdapat error yang menyatakan bahwa library belum terinstall.
Semoga membantu.
